env_config.rb

```diff
@@ -61,6 +60,11 @@
     mandatory :WCA_REGISTRATIONS_BACKEND_URL, :string
   end
 
+  if is_compiling_assets
```

Comment:
Why only when compiling assets? Shouldn't these be available during runtime as well?

Reply:
no, they are only defined in .env.assets.{environment} which is only loaded during asset compilation.

Reply:
Huh, then I'm afraid you have to explain to me why the polling URL is only a known env variable during asset compilation and not during runtime. How is the frontend supposed to call the Polling Lambda during runtime if the URL is not defined?!

Reply:
Because the frontend is already compiled during runtime!